talent/one_on_one_guide.md

```diff
@@ -0,0 +1,99 @@
+**source{d} One-On-One’s Guide**
```

Comment:
Can we use markdown headers? # for level 1, ## for level 2 and so until level 6. This applies to the rest of the document.